#BE-3103 Docs Update - Release Candidate Publish Flow Rules (#503)
appcircleio/appcircle-docusaurus: docs/publish-module/publish-information/marking-release-candidates.md

In order to execute a flow, it must be marked as a Release Candidate (RC). If it is not marked as RC, it cannot be executed.
:::

<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3103-norc.png' />

The chosen version will be visibly distinguished, allowing users to easily identify it as a `Release Candidate`.

<Screenshot url='https://cdn.appcircle.io/docs/assets/publish-release-candidate.png' />

:::tip

> If an app version was configured as `Auto Publish=On` in its build configurations within the build module before it was distributed to the Publish profile, the app version will automatically be marked as `Release Canditate` and Publish Flow operation will be executed directly.

Reviewer: ```suggestion
If you configure an app version with `Auto Publish=On` in its build configurations within the build module before distribution to the Publish profile, Appcircle will automatically mark the app version as a `Release Candidate` and execute the Publish Flow operation directly.
```
